warehouse/models/mart/payments/_payments.yml

```diff
@@ -119,3 +119,54 @@ models:
       - name: transaction_date_pacific
       - name: day_of_week
         description: Day of week value of the transaction in the range [1,7] with Sunday as the first day of the week.
+  - name: fct_elavon__transactions
+    description: Transactions processed by Elavon
```

Comment:
Are there tests we can have here? If there's not a natural primary key can we make a synthetic one?